dev-requirements.txt

```diff
@@ -8,7 +8,6 @@ pytest==7.4.0
 pytest-cov==4.1.0
 pytest-pep8==1.0.6
 pytest-xdist==3.3.1
-selenium==4.26.0
```

Comment:
Since `selenium` is already included in `requirements.txt`, there is no reason to have it here.